frontend/pages/_app.tsx

```diff
@@ -1,4 +1,4 @@
-import 'material-symbols';
+import 'material-symbols/rounded.css';
```

Comment:
Lass uns ausmachen, das wir bei globalen imports dazu schreiben wozu? Das checkt man später vllt nicht mehr.
```suggestion
/** Imports all material symbols globally which we use as our icon pack */
import 'material-symbols/rounded.css';
```